certification.md

```diff
@@ -1,62 +1,49 @@
+| **Receive multi-space, multi-rate restriction push** | Required | 5+ Spaces (3 Rooms, 1 Dorm and 1 Bed) and 2 Rates are mapped and pushed to the [Update Restrictions](channel-manager-api.md#update-restrictions) endpoint. | The channel manager returns ```{ "success": true}``` and data is updated in the channel manager extranet. | Mapping bed and dorm space types is recommended, not required. |
+| **Unmapped Space code error** | Required | An availability push with an unmapped Space type code is sent from the the [Test Hotel](mews-api.md#test-environment) to the [Update Availability](channel-manager-api.md#update-availability) endpoint. | The channel manager returns an error in [plain response](general-remarks.md#plain-response) that includes the unmapped Space type code. |     |
+| **Unmapped Rate code error** | Required | A rate push with an unmapped Rate plan code is sent from the the [Test Hotel](mews-api.md#test-environment) to the [Update Prices](channel-manager-api.md#update-prices) endpoint. | The channel manager returns an error in [plain response](general-remarks.md#plain-response) that includes the unmapped Rate plan code. |    |
+| **Connection inactive or not set up error** | Required | The channel manager connect is deactivated from the channel manager extranet and then an full inventory push is sen from the [Test Hotel](mews-api.md#test-environment) to the channel manager's endpoints. | The channel manager returns an error in [plain response](general-remarks.md#plain-response) that states the channel manager's connection is inactive. |    |
+| **Receive product (package) rates** | Optional | Package Rate and Breakfast product are mapped from the [Test Hotel](mews-api.md#test-environment). Then, a rate push with an package Rate plan code is sent from the the [Test Hotel](mews-api.md#test-environment) to the [Update Prices](channel-manager-api.md#update-prices) endpoint. | The channel manager returns ```{ "success": true}``` and data is updated in the channel manager extranet. |    |
+| **Send a single reservation** | Required | A reservation push with [test credit card](mews-api.md#test-credit-cards) information in the [`paymentCard`](mews-api.md#payment-card) object, an [`adultCount`](mews-api.md#reservation) of 2, a [`childCount`](mews-api.md#reservation) of 1, and a companion in the `guests` property of the booking is sent to the [Process Group](mews-api.md#process-group) endpoint. | The channel manager receives a ```javascript { "clientToken":"[Mews Client Token]", "connectionToken":"[Token of a concrete connection]", "channelManagerId":"[Id sent in the reservation push]", "reservations":[ {"code":"[Code sent in the reservation push]", "confirmationNumber":"[Mews confirmation number]"}]}``` push to the [Confirm Booking](channel-manager-api.md#confirm-booking) endpoint. | The [`paymentCard`](mews-api.md#payment-card) object is required for all reservation pushes in the live environment. |
```

Comment:
can you separate it here to 2nd table?

Inventory push tests
...

Group processing rests
...

Those tables are huge, better separate them so it won't put them off just by looking at it?